Added Skycle.app in community projects (#229)
bluesky-social/atproto-website: content/community/projects.md

## Other Tools

- Skylink [Chrome](https://skylinkchrome.com) and [Firefox](https://skylinkff.com/) extension to detect DIDs while browsing the web, by [@adhdjesse.com](https://bsky.app/profile/did:plc:f55kfczvcsjlaota4ep2xvhx)

> - [Skycle.app](https://skycle.app) - Make your friends circles (as Chirpty) and modify it with your favorite colors, then share it on Bluesky, by [@pirmax.fr](https://bsky.app/profile/pirmax.fr)

Reviewer: ```suggestion
- [Skycle.app](https://skycle.app) - visualize your circle of friends on Bluesky, by [@pirmax.fr](https://bsky.app/profile/pirmax.fr)
```
```suggestion
- [Skycle.app](https://skycle.app) - visualize your circle of friends on Bluesky, by [@pirmax.fr](https://bsky.app/profile/pirmax.fr)
```
